packages/elements/src/utils/fetcher/fetcher.ts

```diff
@@ -17,6 +17,63 @@ export class FetchError extends Error {
   }
 }
 
+export const fetcherWithRawlUrl = async <T, B>({
```

Comment:
Would you mind adding a comment to each of these functions to explain why I would use this fetcher over the main one. Use case etc.

Reason being, we distribute the functions in NPM and some documentation is helpful

Reply:
Typo, should be `fetcherWithRawlUrl`